src/foam/nanos/theme/Theme.js

```diff
@@ -102,157 +112,244 @@ foam.CLASS({
     {
       class: 'Color',
       name: 'primary1',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
```

Comment:
Wouldn't it be better to set these to the values in the ["FOAM default" theme](https://github.com/foam-framework/foam2/blob/master/src/themes)?

Reply:
Nope. Remember that foam exists without nanos so we shouldn't be doing things that require the use of nanos.

Reply:
Right, but why does that make `"unset"` a more reasonable default here than `"#202341"`?

Reply:
Oh sorry misunderstood your previous comment. I'd rather have the defaults than some arbitrary blue colour. My personal use case is that I am embedding foam widgets in a webpage and I want to just inherit the various styles from the parent node by default.

Reply:
Ah, I didn't know `unset` was a CSS thing. I see now. Thanks.